src/EditorFeatures/TestUtilities/Workspaces/TestHostProject.cs

```diff
@@ -327,7 +330,8 @@ public ProjectInfo ToProjectInfo()
                 this.AnalyzerReferences,
                 this.AdditionalDocuments.Select(d => d.ToDocumentInfo()),
                 this.IsSubmission,
-                this.HostObjectType);
+                this.HostObjectType)
+                .WithDefaultNamespace(this.DefaultNamespace);
```

Comment:
i don't understand this.  why not just pass the default namespace into ProjectInfo.Create?

Reply:
Because using the overload with default argument value requires changes to public API. Tracked by https://github.com/dotnet/roslyn/issues/30932